docs/transformation-modules/api/python-api.md

```diff
@@ -0,0 +1,108 @@
+---
+id: python-api
+title: Transformations Python API
+sidebar_label: Python API
+---
+
+This complementary API documentation for `mgp.py`  which contains definitions of the public Transformation Python API provided by Memgraph.
+At the core, this is a wrapper around the **[C API](c-api.md)**. This source file can be found in the Memgraph installation directory,
+under `python_support`. On the standard Debian installation, this will be under `/usr/lib/memgraph/python_support`.
+
+:::caution
+**NOTE:** This part of the documentation is still under development. An updated version will soon be available.
+:::
+
+## `mgp.transformation(func)`
+Register func as a transformation of the current module.
+
+transformation is meant to be used as a decorator function to register a transformation 
+within the current module. The registered func needs to be a callable which either accepts
+a single argument of type Messages or two arguments of type TransCtx and Messages respectively. 
+The return type is fixed and it should be of the form:
+Record(query=str, parameters=mgp.Map). Multiple records can be
+produced by returning an iterable of them. Registering generator functions
+is currently not supported.
```

Comment:
```suggestion
`mgp.transformation` is meant to be used as a decorator function to register a transformation 
within the current module. The registered `func` needs to be a callable which either accepts
a single argument of type `Messages` or two arguments of type `TransCtx` and `Messages` respectively. 
The return type is fixed and it should be of the form:
`Record(query=str, parameters=mgp.Map)`. Multiple records can be
produced by returning an iterable of them. Registering generator functions
is currently not supported.
```

And do the same every time with the backtick when you see something which is code specific in the middle of the text.